### PR TITLE
Parameterized Headline in ChoiceBoard

### PR DESCRIPTION
**Pull request by vparine**
@tiagostutz Here is the PR for parameterizing Headline in Choice board. Could you please review once?

P.S: This is my first ever PR for Open Source. Please let me know if I have gone wrong some where :)

### CODE REVIEW

**Comment by tiagostutz** [comment]
Hi @vparine !! So nice to have your first PR! Awesome, great achievement on that!

I'm reviewing the PR right now. Thanks for your contribution.!

**Comment by vparine** [comment]
@tiagostutz Please let me know if the changes look good or should I work on something?

**Comment by tiagostutz** [comment]
Hi @vparine have you check my review comments?

**Comment by vparine** [comment]
@tiagostutz Somehow I don't see any review comments. Could you please point me to them?

**Comment by tiagostutz** [comment]
> @tiagostutz Somehow I don't see any review comments. Could you please point me to them?

Sure, @vparine ! 
I 've just submitted it again. Please, check whether this appears to you now.

File: app/src/routes/ChoiceBoard.tsx [hunk 1]
```diff
@@ -51,6 +51,7 @@ function ChoiceBoard() {
   };
 
   const dismissButtonLabel = "I don't like either 🤔. Next, please!";
+  var headLine = "Which one would you like to choose?";
```
[the remark this answers is not in the document]
Comment (vparine) [reply app/src/routes/ChoiceBoard.tsx:54]: @tiagostutz It's clear now. I will not be able to work on these changes. Could you please unassign me?

[the remark this answers is not in the document]
Comment (tiagostutz) [reply app/src/routes/ChoiceBoard.tsx:54]: Sure!
Thanks for replying.

**Pull request closed without merging**